docs/guide/dev/browsers/README.md

```diff
@@ -50,27 +50,29 @@ Using both modes is highly unrecommended, because they put the browser in legacy
+CKEditor 4 is also compatible with **Chrome for Android** (which is preinstalled on a growing number of Android devices), though in this case usability also needs to be improved.
 
-<info-box hint="">Prior to version 4.5 CKEditor was disabled in some versions of Chrome for Android and may be disabled in Internet Explorer Mobile for Windows Phone. You can, however, {@link guide/dev/deep_dive/unsupported_environments/README change CKEditor settings to accept any mobile environment} (at your own risk).</info-box>
+<info-box hint="">
+  Prior to version 4.5 CKEditor was disabled in some versions of Chrome for Android and may be disabled in Internet Explorer Mobile for Windows Phone. You can, however, {@link guide/dev/deep_dive/unsupported_environments/README change CKEditor settings to accept any mobile environment} (at your own risk).
+</info-box>
 
 Full mobile support will be introduced in **CKEditor 5**. We aim to have perfect CKEditor support for most popular mobile platforms, so if you encountered an issue with an environment that is unsupported as of now, please report it on our [GitHub issues page](https://github.com/ckeditor/ckeditor4/issues).
 
 ## Unsupported Environments
 
 Please note that Internet Explorer 6 is not supported for CKEditor 4. CKEditor 4.1.3 is the last release to support Internet Explorer 7 and Firefox 3.6.
 
-When CKEditor detects an unsupported environment, it will simply not load and the user will be presented with a standard editable element (for example a `<textarea>` or `<div>`) that the editor was supposed to replace.
+When CKEditor 4 detects an unsupported environment, it will simply not load and the user will be presented with a standard editable element (for example a `<textarea>` or `<div>`) that the editor was supposed to replace.
 
 It is possible to skip the compatibility check and {@link guide/dev/deep_dive/unsupported_environments/README enable CKEditor in all environments}, including the unsupported ones, but this is an experimental feature aimed at more advanced developers and is highly unrecommended in production environments.
 
-**Note:** Prior to version 4.5 CKEditor was disabled in all environments which were not recognized as compatible. Since CKEditor 4.5 the whitelist was changed to a blacklist, so currently CKEditor is only disabled in environments which it is known to be incompatible with (for example Internet Explorer 7 and below and Firefox 3.6 and below). Read more in {@linkapi CKEDITOR.env.isCompatible CKEDITOR.env.isCompatible}
+**Note:** Prior to version 4.5 CKEditor was disabled in all environments which were not recognized as compatible. Since CKEditor 4.5 the whitelist was changed to a blacklist, so currently CKEditor 4 is only disabled in environments which it is known to be incompatible with (for example Internet Explorer 7 and below and Firefox 3.6 and below). Read more in {@linkapi CKEDITOR.env.isCompatible CKEDITOR.env.isCompatible}
```

Comment:
```suggestion
**Note:** Prior to version 4.5 CKEditor was disabled in all environments which were not recognized as compatible. Since CKEditor 4.5 the whitelist was changed to a blacklist, so currently CKEditor 4 is only disabled in environments which it is known to be incompatible with (for example Internet Explorer 7 and below and Firefox 3.6 and below). Read more in {@linkapi CKEDITOR.env.isCompatible CKEDITOR.env.isCompatible}.
```